src/routes/projects.rs

```diff
@@ -551,6 +552,42 @@ pub async fn project_edit(
                     }
                 }
 
+                if team_member.map(|x| !x.accepted).unwrap_or(true) {
+                    let user_id: database::models::ids::UserId = user.id.into();
+                    let notified_members = sqlx::query!(
+                        "
+                        SELECT tm.user_id id
+                        FROM team_members tm
+                        WHERE tm.team_id = $1 AND tm.user_id <> $2 AND tm.accepted
```

Comment:
Could we use `!=` instead of `<>` here? it's more readable and consistent w/ the codebase. Also, does the check even need to be there in the first place? Don't see a case in this code path where it would send a notif to the team member who set it (since it only sends notifs if team members haven't sent it)